### PR TITLE
Fix management commands compatibility with Django 1.10

### PR DESCRIPTION
**Pull request by m1kola**
See #2723. ~~Incorporates #2724~~

### CODE REVIEW

File: wagtail/wagtailcore/management/commands/move_pages.py [hunk 1]
```diff
@@ -6,16 +6,15 @@
...
 class Command(BaseCommand):
-    args = "<from id> <to id>"
-
-    def handle(self, _from_id, _to_id, **options):
-        # Convert args to integers
-        from_id = int(_from_id)
-        to_id = int(_to_id)
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument('from_id', nargs='?', type=int)
```
Comment (timgraham) [inline wagtail/wagtailcore/management/commands/move_pages.py]: I think you want to drop `nargs='?'` in all cases. Aren't these required arguments? `nargs='?'` makes it an optional argument defaulting to `default` (`None` in this case).

**Comment by m1kola** [comment]
@timgraham, thank you for review!

PR was rebased and updated according comment.

**Comment by timgraham** [comment]
LGTM

**Comment by gasman** [comment]
Merged in b54b2d837c0519eb08f4e44458669965d5625de6 - thanks!

**Pull request closed without merging**